### DISCUSSION TITLE
`docker attach` integration

### DISCUSSION THREAD

**Discussion by Seercat3160** [louislam/dockge#261] [Feature Request]
### ⚠️ Please verify that this feature request has NOT been suggested before.

- [X] I checked and didn't find similar feature request

### 🏷️ Feature Request Type

UI Feature

### 🔖 Feature description

A system, similar to the `bash` terminal, where a web terminal can be used to attach to a running container in a stack.
It would also be useful if a container could be set such that the stdin/out/err streams are logged by Dockge in the background so when the container is attached to, there is a history buffer rather than just a blank terminal.

### ✔️ Solution

I would like to be able to click a button on a container, like the button which opens the bash terminal in the container, and have a full-page console opened which displays the container's stdin/out/err as is done when running `docker attach` from the CLI on a machine.

### 📝 Additional Context

This would be useful for running containers such as docker-minecraft-server, where having direct access to the container's console is of great benefit.
Prior art in this space: Pterodactyl Panel, which works similarly to allow web management of game servers in Docker containers.

**Comment by sveken** [comment]
Only just noticed this feature was missing as i was writing a guide to host our game docker image using dockge. 

However once the game server is up, to issue any console commands requires the attach feature. I could only find how to shell into the base container. Not attach to the running program like we can in portainer.
